Frameworks/Foundation/NSFileManager.mm

```diff
@@ -91,6 +94,20 @@
 NSString* const NSFileProtectionCompleteUnlessOpen = @"NSFileProtectionCompleteUnlessOpen";
 NSString* const NSFileProtectionCompleteUntilFirstUserAuthentication = @"NSFileProtectionCompleteUntilFirstUserAuthentication";
 
+static const winrt::hstring sc_fileSizeKey = L"System.Size";
+static const winrt::hstring sc_fileAttributesKey = L"System.FileAttributes";
+static const winrt::hstring sc_fileModificationDateKey = L"System.DateModified";
+static const winrt::hstring sc_fileCreationDateKey = L"System.DateCreated";
+
+// Conversion math from https://msdn.microsoft.com/en-us/library/ms724228
+static DateTime __NSDateToSystemTime(NSDate* date) {
+    return { (int64_t)([date timeIntervalSince1970] * 10000000) + 116444736000000000 };
```

Comment:
There's a private func in CFPlatform.c but that's pretty inaccessible so whatever